<!DOCTYPE HTML>
<!--
	Phantom by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Generic - Phantom by HTML5 UP</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<div class="inner">

							<!-- Logo -->
								<a href="index.html" class="logo">
									<span class="symbol"><img src="images/logo.svg" alt="" /></span><span class="title">Kira Mathias-Prabhu | Portfolio</span>
								</a>

							<!-- Nav -->
								<nav>
									<ul>
										<li><a href="#menu">Menu</a></li>
									</ul>
								</nav>

						</div>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<h2>Menu</h2>
						<ul>
							<li><a href="index.html">Portfolio</a></li>
							<li><a href="generic.html">About Me</a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">
						<div class="inner">
							<h1>Research with the Systems Biology Group</h1>
							<h3>In Brief</h3>
							<p>The goal of this project was automated retrieval of single-cell RNA expression data and extraction of source cell type information in order to start a user-friendly scRNA database, and develop a cell type classifier for novel sequence data. Results of the project include <a href="https://scquery.cs.cmu.edu">scQuery</a>, and the corresponding paper, <a href="https://www.nature.com/articles/s41467-018-07165-2/">A web server for comparative analysis of single-cell RNA-seq data</a>.</p>
							<h3>Project Members</h3>
							<p>I worked with Dr. Matt Ruffalo (a doctoral student at the time), under the supervision of Dr. Ziv Bar-Joseph as part of the Systems Biology Group in the Computational Biology Department at Carnegie Mellon.</p>
							<h3>Project Contribution</h3>
							<p>I worked on query development for data aggregation and extraction of cell type information for the database development part of this project. I first generated and tested queries to download single-cell RNA expression data for mice from GEO (Gene Ontology Online). This was done using the Entrez API in Python. I then verified that the data was in fact mouse data, and extracted the cell type from a tag in the file. The result was a map per series (GSE) of gene expression samples (GSM) to cell types</p>
							
						</div>
					</div>

				<!-- Footer -->

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>